pgmpy/models/BayesianNetwork.py

```diff
@@ -279,8 +279,17 @@ def get_cpds(self, node=None):
 
         Returns
         -------
-        A list of TabularCPDs: list
-
+        - If 'node' is specified: 
+            Returns the TabularCPD object corresponding to the specified node.
+            Raises ValueError if the node is not present in the model.
+        - If 'node' is not specified: 
+            Returns a list of all TabularCPD objects that have been added to the model.
+        Raises
+        ------
+        ValueError
+            If the specified node is not present in the model.
+            
+            
         Examples
         --------
         >>> from pgmpy.models import BayesianNetwork
```

Comment:
I would completely switch out this example with something similar to the `get_example_model` example I had added to the issue.